services/ui_backend_service/api/data_refiner.py

```diff
@@ -75,6 +80,9 @@ def __init__(self):
         super().__init__(field_names=["task_ok", "foreach_stack"])
 
     async def postprocess(self, response: DBResponse):
+        if FEATURE_REFINE_DISABLE:
+            return response
+
```

Comment:
if we skip postprocessing completely with feature flags, then the api responses will contain internal fields for f.ex. tasks due to them having s3 locations for refinement (foreach_var / _task_ok)